vws.gemspec

```diff
@@ -20,13 +20,13 @@ Gem::Specification.new do |spec|
 
 
   #development depedencies
-  spec.add_development_dependency "bundler", "~> 1.9.1"
+  spec.add_development_dependency "bundler"
```

Comment:
Bundler version should not be enforced because it is "just" a development dependency.